src/main/java/com/treasuredata/client/TDClientConfig.java

```diff
@@ -243,7 +243,7 @@ public static Properties readTDConf()
         Properties p = new Properties();
         File file = new File(System.getProperty("user.home", "./"), String.format(".td/td.conf"));
         if (!file.exists()) {
-            logger.warn(String.format("config file %s is not found", file));
+            logger.debug(String.format("config file %s is not found", file));
```

Comment:
Since this method is designed to read `$HOME/.td/td.conf`, it is better to print the warning in case of file not found.

Reply:
ok. I'll revert the change.